tiny-lang.cabal

```diff
@@ -18,7 +18,12 @@ library
   exposed-modules:     TinyLang.Prelude,
                        TinyLang.Var
                        TinyLang.Boolean.Core
+                       TinyLang.Boolean.Evaluator
+                       TinyLang.Boolean.Environment
+                       TinyLang.Boolean.Parser
```

Comment:
The actual file doesn't seem to be added.

Reply:
Do you want me to add that? It doesn't quite work at the moment and I'll probably rewrite it using the Expr library.  Maybe you should just take Parser out of the cabal file for now.

Reply:
Yeah, please take it out, otherwise the code will not build.

Reply:
Ah, sorry, misread as "Maybe I should just take Parser". I'll take it out.